source/BP-Essentials/Methods/Check/CheckBanned.cs

```diff
@@ -18,14 +18,20 @@ public static void Run(SvPlayer player)
             {
                 try
                 {
-                    if (player.player.admin || string.IsNullOrEmpty(player.connection.IP.Trim()))
+                    if (player.player.admin || String.IsNullOrEmpty(player.connection.IP.Trim()))
+                    {
                         return;
+                    }
                     foreach (var line in File.ReadAllLines(BansFile))
-                        if (string.IsNullOrEmpty(line) || !line.StartsWith("# " + player.player.username, StringComparison.CurrentCulture))
-                            continue;
-                    Debug.Log($"{PlaceholderParser.ParseTimeStamp()} [WARNING] {player.player.username} Joined while banned! IP: {player.connection.IP}");
-                    player.svManager.AddBanned(player.player);
-                    player.svManager.Disconnect(player.connection, DisconnectTypes.Banned);
+                    {
+                        if (line.Trim() == $"# {player.player.username}".Trim()) || line == player.connection.IP.Trim())
```

Comment:
```suggestion
                        if (line.Trim() == $"# {player.player.username}".Trim() || line == player.connection.IP.Trim())
```

Reply:
Yeah, i just did that without opening visual studio, i know i should post that, just were fixing and didn't check if there was a problem i had to go and i posted that, thanks you ;)